content/en/docs/Monitoring/http-requests.md

```diff
@@ -0,0 +1,33 @@
+---
+title: Http Requests Counts
```

Comment:
HTTP Request Count

Reply:
Fixed

Reply:
"Count" is one of those words which implies plural. It's one counter with numbers that can be greater than zero.